### PR TITLE
Set var name for iD - ID_APPLICATION for web container

### PR DESCRIPTION
**Pull request by Rub21**
cc. @batpad @geohacker

### CODE REVIEW

File: images/web/start.sh [hunk 1]
```diff
@@ -28,7 +28,8 @@ sed -i -e 's/openstreetmap@example.com/'$MAILER_FROM'/g' $workdir/config/setting
 sed -i -e 's/smtp_port: 25/smtp_port: '$MAILER_PORT'/g' $workdir/config/settings.yml
 
 #### SET UP ID KEY
-sed -i -e 's/#id_key: ""/id_key: "'$OPENSTREETMAP_id_key'"/g' $workdir/config/settings.yml
+sed -i -e 's/#id_key: ""/id_key: "'$ID_APPLICATION'"/g' $workdir/config/settings.yml
```
Comment (geohacker) [inline images/web/start.sh]: We dont need this anymore.

File: osm-seed/values.yaml [hunk 2]
```diff
@@ -132,6 +132,7 @@ web:
     OVERPASS_URL: 'overpassApi'
     OAUTH_CLIENT_ID: 'abcd' # Create client ID and key as OAuth2 Application
     OAUTH_KEY: 'xyz'
+    ID_APPLICATION: 'xyz'
```
Comment (geohacker) [inline osm-seed/values.yaml]: Is this needed here?

Comment (Rub21) [reply osm-seed/values.yaml]: yes, to pass as an example and  to show what env var are required.

**Comment by geohacker** [comment]
@Rub21 I think we could close this PR without merging for the time being as these changes are not required. What do you think?

**Comment by Rub21** [comment]
Closing this PR,  because of OHM-iD version is different from osm-seed-id version.

**Pull request closed without merging**